### ISSUE TITLE
V7: Increased bracket height

### ISSUE THREAD

**Issue by chmielcode** [subframe7536/maple-font#192]
In recent V7 releases bracket height has increased 

before
![obraz](https://github.com/subframe7536/maple-font/assets/45938987/88f4a84e-4ab4-4e0b-887f-bd93b3244496)

now (b18)
![obraz](https://github.com/subframe7536/maple-font/assets/45938987/5e3607b9-9cb4-4183-a0a4-58ca26a89cc6)

The "now" version is better in one way (all brackets have the same height) and worse in the other (brackets are too big). This is of course my opinion, but brackets crossing underline in apps like Big Query Studio wasn't an issue before.

I mean it still looks good, but bracket started to stand out here and there so I've decided to create this issue mostly to confirm it was intentional.

**Comment by chmielcode** [comment]
I did and it makes all characters visually slightly thinner and (excluding brackets) taller.

Was
![obraz](https://github.com/subframe7536/maple-font/assets/45938987/ad3199c3-74c9-42a7-a389-f87a0de142a3)
Latest relaeses
![obraz](https://github.com/subframe7536/maple-font/assets/45938987/3576e297-a9c3-4bdf-8aad-87ce2a3a6950)
The one you provided here
![obraz](https://github.com/subframe7536/maple-font/assets/45938987/1800b038-bb21-4fc1-abbf-124f6225b21c)

Other characters:
![obraz](https://github.com/subframe7536/maple-font/assets/45938987/eed415ab-68f2-4241-a4d2-8a3ca8a71668)
vs ttf with bracket fix
![obraz](https://github.com/subframe7536/maple-font/assets/45938987/ad0f9622-98e6-46e4-a555-2ba312ebdc4f)

In short this fix makes brackets appear a little smaller but still too big. It also makes all other characters lose their (much desired) thickness. If there is no better way to do it, I'd go with bigger brackets all the way :)

**Comment by subframe7536** [comment]
The thickness maybe caused by the font hint. The one I provide has no hint at all.

Also, the character size is similar to other popular fonts like `Fira Code` or `Jetbrains Mono`

**Comment by subframe7536** [comment]
The thickness maybe caused by the font hint. The one I provide has no hint at all.

Also, the character size is similar to other popular fonts like `Fira Code` or `Jetbrains Mono`

**Comment by subframe7536** [comment]
release in v7.0-beta19
